doc/observable.md

```diff
@@ -259,19 +259,19 @@ Conclusion:
 
 ## Anatomy of an Observable
 
-Observables are **created** using `Rx.Observable.create` or a creation operator, are **subscribed** to with an Observer, **execute** to deliver `next` / `error` / `complete` notifications to the Observer, and their execution may be **disposed**. These four aspects are all encoded in an Observable instance, but some of these aspects are related to other types, like Observer an Subscription.
+Observables are **created** using `Rx.Observable.create` or a creation operator, are **subscribed** to with an Observer, **execute** to deliver `next` / `error` / `complete` notifications to the Observer, and their execution may be **disposed**. These four aspects are all encoded in an Observable instance, but some of these aspects are related to other types, like Observer and Subscription.
 
 Four core Observable concerns:
-- **Creation**
+- **Create**
 - **Subscribe**
-- **Execution**
-- **Disposal**
+- **Execute**
+- **Dispose**
 
```

Comment:
I completely get that--but only after getting this far in with RxJS. Someone fresh reading the manual isn't so aware of which terms are Types and which are aliases, like Subscriber, which is not a true type.

For the same reason @blesh doesn't like `I am concerned with create`, `I am concerned with subscribe` is not a proper sentence. Maybe this section regarding "concerns" should be contain fuller sentences:

The four primary concerns are:

**Creating** Observables
What happens when an observer **subscribes** to an Observable.
**Executing** notifications.
What to do when an observer **disposes** a (or **unsubscribes** from) a subscription.